src/index.ts

```diff
@@ -159,23 +164,30 @@ export default class Client extends BaseClient {
       avs_pb.ListWalletReq
     >("listWallets", request, options);
 
-    return result.getWalletsList().map((item) => item.toObject());
+    return result.getItemsList().map((item) => item.toObject());
   }
 
-  async createWallet(
-    { salt, factoryAddress }: CreateWalletReq,
+  /**
+   * Add a new smart wallet address to the wallet list
+   * @param {string} salt - The salt for the wallet
+   * @param {string} factoryAddress - Factory address for the wallet
+   * @param {RequestOptions} options - Request options
+   * @returns {Promise<SmartWallet>} - The added SmartWallet object
+   */
+  async addWallet(
```

Comment:
I’m experimenting with the name `addWallet` for adding a new SmartWallet to the list. It’s `avs_pb.GetWalletReq()` behind the scene.